demos/Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN apt-get update && apt-get install -y \
     && docker-php-ext-install intl \
     && docker-php-ext-install pdo pdo_mysql
 
+RUN curl -sL https://deb.nodesource.com/setup_12.x  | bash - \
+    && apt-get update && apt-get -y install nodejs \
+    && npm install -g npm
```

Comment:
@ibelar Can we remove `npm` from line above (`libicu-dev git jq unzip npm`) if installed here?

Reply:
this is not for installing it but for updating it.

Reply:
The dist of the used image is conservative... After the sources are updated, I would expect we can install latest straight as I have advised above. I do not see any advantage why to install not-wanted version first.